### PR TITLE
Update README.md for LCM with LoRA

### PR DESCRIPTION
**Pull request by zhuo-yoyowz**
Update README.md for LCM with LoRA

### CODE REVIEW

File: image_generation/lcm_dreamshaper_v7/cpp/README.md [hunk 1]
```diff
@@ -95,15 +112,15 @@ Positive prompt: a beautiful pink unicorn
 
 Read the numpy latent input and noise for scheduler instead of C++ std lib for the alignment with Python pipeline.
 
-* Generate image with random data generated by Python: `./build/lcm_dreamshaper -r`
+* Generate image with random data generated by Python: `./build/Release/lcm_dreamshaper -r`
```
Comment (ilya-lavrenov) [inline image_generation/lcm_dreamshaper_v7/cpp/README.md:115]: `Release` is required only for multi config cmake generators like MSVC or Ninja Multi-Config
https://cmake.org/cmake/help/latest/prop_gbl/GENERATOR_IS_MULTI_CONFIG.html

File: image_generation/lcm_dreamshaper_v7/cpp/README.md [hunk 2]
```diff
@@ -1,26 +1,46 @@
...
+
 The pure C++ text-to-image pipeline, driven by the OpenVINO native API for SD v1.5 Latent Consistency Model with LCM Scheduler. It includes advanced features like LoRA integration with safetensors and [OpenVINO Tokenizers](https://github.com/openvinotoolkit/openvino_tokenizers). Loading `openvino_tokenizers` to `ov::Core` enables tokenization. [The common folder](../../common/) contains schedulers for image generation and `imwrite()` for saving `bmp` images. This demo has been tested for Linux platform only. There is also a Jupyter [notebook](https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/263-latent-consistency-models-image-generation/263-lcm-lora-controlnet.ipynb) which provides an example of image generaztion in Python.
 
 > [!NOTE]
 > This tutorial assumes that the current working directory is `<openvino.genai repo>/image_generation/lcm_dreamshaper_v7/cpp/` and all paths are relative to this folder.
 
 ## Step 1: Prepare build environment
 
-Prerequisites:
-- Conda ([installation guide](https://conda.io/projects/conda/en/latest/user-guide/install/index.html))
```
Comment (ilya-lavrenov) [inline image_generation/lcm_dreamshaper_v7/cpp/README.md:10]: why conda is not used?

Comment (yangsu2022) [reply image_generation/lcm_dreamshaper_v7/cpp/README.md:10]: `conda install -c conda-forge openvino` will automatically print and set env var with `SET "VS_YEAR=2019"`, while only VS2022 exists on my Win11.
The details in the attachment. [log.txt](https://github.com/user-attachments/files/15788970/log.txt)
Shall we align with the previous text generation without conda?

File: image_generation/lcm_dreamshaper_v7/cpp/README.md [hunk 3]
```diff
@@ -1,26 +1,46 @@
...
+```
+
+NOTE: If you are using Windows, use `lcm_cpp_env\Scripts\activate`command instead.
+
+### Cloning the Repository
+
+To clone the repository, run the following command:
+
+```shell
+git clone https://github.com/openvinotoolkit/openvino.genai.git
```
Comment (Wovchena) [inline image_generation/lcm_dreamshaper_v7/cpp/README.md:37]:
```suggestion
git clone --recursive https://github.com/openvinotoolkit/openvino.genai.git
```
So you can remove `git submodule update --init` below

File: image_generation/lcm_dreamshaper_v7/cpp/README.md [hunk 4]
```diff
@@ -31,9 +51,7 @@ conda env config vars set LD_LIBRARY_PATH=$CONDA_PREFIX/lib:$LD_LIBRARY_PATH
 
     ```shell
     git submodule update --init
-    conda activate openvino_lcm_cpp
     python -m pip install -r requirements.txt
-    python -m pip install ../../../thirdparty/openvino_tokenizers/[transformers]
```
Comment (Wovchena) [inline image_generation/lcm_dreamshaper_v7/cpp/README.md:36]: Why is this line removed?

`optimum-cli` needs `openvino_tokenizers`. I know that `requirements.txt` install `openvino_tokenizers`, but it may provide an outdated version compared to our submodule.

File: image_generation/lcm_dreamshaper_v7/cpp/README.md
Comment (Wovchena) [inline image_generation/lcm_dreamshaper_v7/cpp/README.md:1]: Please apply the same to [stable_diffusion_1_5/cpp](https://github.com/openvinotoolkit/openvino.genai/tree/master/image_generation/stable_diffusion_1_5/cpp) to keep them aligned.

Comment (Wovchena) [inline image_generation/lcm_dreamshaper_v7/cpp/README.md:1]: It's not required for this PR to be merged, but  it would be nice if you extended tests to cover the scenario described in the proposed version of README.md. The tests are in https://github.com/openvinotoolkit/openvino.genai/blob/master/.github/workflows/lcm_dreamshaper_cpp.yml and https://github.com/openvinotoolkit/openvino.genai/tree/master/image_generation/lcm_dreamshaper_v7/cpp. Please, don't remove conda tests

**Review by Wovchena — CHANGES_REQUESTED** [review]

**Comment by yatarkan** [comment]
Closing in favor of #761 and #772
